geosnap/data/data.py

```diff
@@ -675,6 +675,31 @@ def store_ncdb(filepath):
 
     df = df.set_index("geoid")
 
+    #### Beginning of New Code ####
+
+    year = df["year"]
+
+    # Inflation logic for ncdb.
+    inflate_cols = [
+        "MDVALHS",
+        "MDGRENT",
+        "MDHHY",
+    ]
+    # Five rows have missing ncdb labels in variables.csv.
+    # per capita income, missing
+    # median household income white, missing
+    # median household income black, missing
+    # median household income hispanic, missing
+    # median household income asian, missing
+
+    inflate_available = list(set(df.columns).intersection(set(inflate_cols)))
+
+    if len(inflate_available):
+        df = adjust_inflation(df, inflate_available, year)
+    return df
+
+    #### End of New Code ####
```

Comment:
this will need to work a little bit differently from the ltdb example. Unlike in the `store_ltdb` function, we've only got a single dataframe here, so we need to slice the ncdb dataframe by year and apply the `adjust_inflation` to each slice

Reply:
Ok, I will take a look.